public/app/features/alerting/unified/RuleList.test.tsx

```diff
@@ -121,16 +124,19 @@ const ui = {
   newRuleButton: byRole('link', { name: 'New alert rule' }),
 
   editGroupModal: {
-    namespaceInput: byLabelText('Namespace'),
-    ruleGroupInput: byLabelText('Rule group'),
-    intervalInput: byLabelText('Rule group evaluation interval'),
+    namespaceInput: byRole('textbox', { hidden: true, name: /namespace/i }),
+    ruleGroupInput: byRole('textbox', { name: 'Evaluation group', exact: true }),
+    intervalInput: byRole('textbox', {
+      name: /Rule group evaluation interval Evaluation interval should be smaller or equal than For values for existing rules in this group./i,
+    }),
```

Comment:
```suggestion
      name: /Rule group evaluation interval Evaluation interval should be smaller or equal to For values for existing rules in this group./i,
    }),
```

Reply:
Do you need a full stop or a break between the name Rule group evaluation interval and the description? Otherwise it reads as a full sentence.

Reply:
Maybe this way you can understand better what I mean:
`name: /Rule group evaluation interval Evaluation interval should be smaller or equal than 'For' values for existing rules in this group.`